Source/Dafny/ConcreteSyntax/ConcreteSyntaxTreeUtils.cs

```diff
@@ -55,15 +55,29 @@ public static class ConcreteSyntaxTreeUtils {
       switch (open) {
         case BraceStyle.Space:
           outer.Write(" ");
+          outer.WriteLine("{");
           break;
         case BraceStyle.Newline:
           outer.WriteLine();
+          outer.WriteLine("{");
+          break;
+        case BraceStyle.Pindent:
+          outer.WriteLine();
+          PythonCompiler.indent += 1;
           break;
       }
 
-      outer.WriteLine("{");
-      body = outer.Fork(1);
-      outer.Write("}");
+
+      body = outer.Fork((open == BraceStyle.Pindent) ? PythonCompiler.indent : 1);
```

Comment:
I'm confused, why doesn't 1 work? The `1` is relative to the existing indentation right?

Reply:
At first I didn't know about the _relativeIndentLevel_ that's why initiated _PythonCompiler.indent_ and was using that, despite the name says it's relative I assumed it being global and added _PythonCompiler.indent_ in the equation there. Thank for the input I will make changes to that.